packages/experiments/src/components/Card/CardFrame/CardFrame.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { ICardFrameProps, ICardFrameStyles, ICardDropDownOption } from './CardFrame.types';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { getStyles } from './CardFrame.styles';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { IOverflowSetItemProps, OverflowSet } from 'office-ui-fabric-react/lib/OverflowSet';
+
+export class CardFrame extends React.Component<ICardFrameProps, {}> {
+  constructor(props: ICardFrameProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const getClassNames = classNamesFunction<ICardFrameProps, ICardFrameStyles>();
+    const { fontFamily, fontSize, cardTitle, seperatorColor, titleTextColor } = this.props;
+    const classNames = getClassNames(getStyles, { cardTitle, fontFamily, fontSize, seperatorColor, titleTextColor });
+    const overflowItems: ICardDropDownOption[] | undefined = this.props.cardDropDownOptions;
+    const cardDropDownOptions: IOverflowSetItemProps[] = [];
+    if (overflowItems !== undefined) {
+      overflowItems.map((data: ICardDropDownOption, index: number) => {
+        const cardDropDownOption: IOverflowSetItemProps = {
+          key: index.toString(),
+          name: data.name,
+          icon: data.icon,
+          ariaLable: data.ariaLabel ? data.ariaLabel : data.name,
```

Comment:
>ariaLable [](start = 10, length = 9)

spelling?

Reply:
Fixed

---
In reply to: [195604678](https://github.com/OfficeDev/office-ui-fabric-react/pull/5196#discussion_r195604678) [](ancestors = 195604678)